src/elm/Styles/Colours.elm

```diff
@@ -0,0 +1,78 @@
+module Styles.Colours exposing (getPastel)
+
+import Styles.Classes exposing (tachs)
+import Color exposing (..)
+import Array exposing (..)
+import Maybe exposing (withDefault)
+
+
+type alias Swatches =
+    Array RgbString
+
```

Comment:
Is there any particular reason you're using the `Array` type here? Why use it over `List`

Also, is each `RgbString` going to be unique? If they are it might be worth considering [`Set`](http://package.elm-lang.org/packages/elm-lang/core/5.1.1/Set) instead of `Array`

This again may be overoptimizing but fun to look at and learn about at least 😄